components/summary/summary.hbs

```diff
@@ -1,18 +1,16 @@
 <div class="summary">
-      <h2 class="summary__title saturn" id="">Summary - Section Title</h2>
-<!-- Summary -->
-      <div class="summary__block">
-        <div class="summary__items">
-          <div class="summary__question" id="">
-             What are the dates of the sales period you are reporting for?
-          </div>
-            <div class="summary__answer">
-              <div class="summary__answer-text" id="">01 January 2015 to 02 February 2017</div>
-              <div class="summary__edit">
-                <a href="#edit" class="summary__edit-link">Edit <span class="u-vh">your answer</span></a>
-              </div>
-            </div>
+  <h2 class="summary__title neptune" id="">Summary - Section Title</h2>
+  <div class="summary__block">
+    <div class="summary__items">
+      <div class="summary__answer">
+        <div class="summary__question" id="">
+           What are the dates of the sales period you are reporting for?
         </div>
-     </div>
-<!-- /Summary -->
+        <div class="summary__answer-text" id="">01 January 2015 to 02 February 2017</div>
+        <div class="summary__edit">
+          <a href="#edit" class="summary__edit-link">Change <span class="u-vh">your answer to "What are the dates of the sales period you are reporting for?"</span></a>
```

Comment:
I think it might be improved if we tried changing:

`Change <span class="u-vh">your answer to "What are the dates of the sales period you are reporting for?"</span>`

**to**

`Change <span class="u-vh">your answer to "What are the dates of the sales period you are reporting for?". The current value is "01 January 2015 to 02 February 2017".</span>`

as this would give the link rotor all of the information that the user requires to make an informed choice in selecting the link. This should be done for all answer types.

Reply:
Would it be possible to know what the question type was and therefore change the "current value" to "The date range is 01 January..." for "What are the dates of the sales period you are reporting for?" and for the Total turnover question it would be "The value is £234,000.00"?